jose.go

```diff
@@ -277,14 +277,19 @@ func (c Claims) Get(name string) (string, bool) {
 
 func CalculateHeadersToPropagate(propagationCfg [][]string, claims map[string]interface{}) (map[string]string, error) {
 	if len(propagationCfg) == 0 {
-		return nil, fmt.Errorf("JOSE: no headers to propagate. Config size: %d", len(propagationCfg))
+		return nil, fmt.Errorf("no headers to propagate. Config size: %d", len(propagationCfg))
```

Comment:
in this case, we should return an exported error, so it can be addressed on the caller